src/pages/blog/starlingx-dual-stack-support.md

```diff
@@ -0,0 +1,277 @@
+- The primary address pool family of a network cannot be modified after network creation, requiring a system reinstall to change.
+
+## API/CLI Interactions
+
+- The `network-addrpool-assign`, `network-addrpool-remove`, `network-addrpool-list`, and `network-addrpool-show` APIs handle network to address pool associations.
+- The `system network-add` command creates the first association.
+- The `system addrpool-modify` command allows for editing all address pool parameters.
+
+## Installation Bootstrap
+
+- Dual-stack installations require specifying secondary subnets in bootstrap variables using comma-separated values. E.g.:
+  - `external_oam_subnet: fd00::/64,10.20.2.0/24`, the order of network listing determines primary and secondary address pools. In the example fd00::/64 is the primary
+- All primary subnets must use the same address family.
+
+## DNS Server Configuration
+- Optionally, name servers can be configured with both IPv4 and IPv6 addresses using the `system dns-modify` command, still limited to 2 server addresses.
+
+## DC Operations
+- Subclouds can be installed in dual-stack mode if their version supports it.
+- All operational communication between the system controller and subclouds uses the primary address pool.
+- The system-controller and subclouds can operate in different network modes but must share the same primary address family in the OAM and management networks (for subclouds this can be OAM and admin networks).
+- Geo redundancy uses the primary pools to communicate.
+
+## Public Endpoints
+- Public OAM endpoints can be accessed via the secondary address using the same L4 port. HAProxy maps external requests to internal endpoints.
+
+## Network Address Modification at Runtime
+- Only the OAM, Admin, and Management networks can be modified using the `addrpool-modify` command during runtime. 
+- Other network’s address pool modifications require reinstallation.
+
+## External OAM API and CLI
+- The external-OAM API, although deprecated, can still be used to modify the OAM network primary pool. 
+- Corresponding CLIs are `system oam-modify` and `system oam-show`.
+
+## Kubernetes in Dual-Stack
+- The OAM, cluster-host, cluster-service, and cluster-pod networks must be configured for dual-stack support in Kubernetes. 
+- Runtime changes trigger quick restarts for the kube-API-server and kube-controller-manager pods. 
+- New pods will automatically receive both primary and secondary addresses when created in dual-stack mode. Existing pods may retain their current primary addresses and require restarts to acquire secondary addresses. 
+
+## Runtime Configuration
+- To add dual-stack to a running system, follow the steps outlined in the guide, which include adding address pools and associating them with networks.
+- Reverting to single-stack involves removing the network association with the address pool.
```

Comment:
```suggestion
## Address Pool Management

As the feature’s name suggests, two address pools are required for dual-stack operation, with the first one linked to the network at creation. This also means that the primary pool cannot be removed later, while removing the second pool can transform the system into a single-stack mode. As the dual-stack support is still new in the platform, there are some additional limitations to this oncfiguration options, for example, the pxeboot network currently only supports IPv4.

## Installation Bootstrap

Dual-stack installations require specifying secondary subnets in bootstrap variables using comma-separated values. The order of network listing determines primary and secondary address pools, for instance, in `external_oam_subnet: fd00::/64,10.20.2.0/24`,  'fd00::/64' is the primary. You will also need to remember that all primary subnets must use the same address family.

## Distributed Cloud Operations

The Distributed Cloud architecture is what differentiates StarloingX from most other cloud platforms, and therefore it has been crucial that it supports the dual-stack configuration option as well. Subclouds can be installed in dual-stack mode if their version is new enough to supports it. When this setuip is used, all operational communication between the system controller and subclouds uses the primary address pool.

It is also important to note, that the System Controller and subclouds can operate in different network modes, however, they must share the same primary address family in the OAM and management networks (for subclouds this can be OAM and admin networks). The Geo redundancy feature also uses the primary pools to communicate.

## Kubernetes Configuration

 When dual-stack mode is used with Kubernetes, the OAM, cluster-host, cluster-service, and cluster-pod networks must be configured for dual-stack support. And be mindful that runtime changes trigger quick restarts for the kube-API-server and kube-controller-manager pods.

Once the system is deployed and configured, new pods will automatically receive both primary and secondary addresses. At the same time, existing pods may retain their current primary addresses and require restarts to acquire secondary addresses. 

## Runtime Configuration

You can also update a running system to use dual-stack, follow the steps outlined in the guide, which include adding address pools and associating them with networks. In the scenario of reverting to single-stack configuration, you will need to remove the network association with the address pool.
```

Reply:
The long bullet point lists make the blog post look a bit less like an article and more of like technical documentation. In order to avoid that, I added a suggestion to keep some highlights while encouraging the readers to consult the feature documentation for further details.